packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -462,7 +462,7 @@ def get_raw_safe_transaction(  # pylint: disable=too-many-arguments,too-many-loc
             tx_parameters.update(ledger_api.try_get_gas_pricing(old_price=old_price))
         # note, the next line makes an eth_estimateGas call iff gas is not set!
         transaction_dict = w3_tx.buildTransaction(tx_parameters)
-        if configured_gas != 1:
+        if configured_gas != MIN_GAS:
```

Comment:
I feel like the name `MIN_GAS` is a bit confusing here. Maybe something like `NO_GAS` or something like that is more appropriate. Also, I don't see why we cannot set `configured_gas` to None and check for it as `None`, that seems much more natural to me.